modules/contact/controllers/update.controller.js

```diff
@@ -0,0 +1,34 @@
+const Joi = require('joi');
+const updateContact = require('../services/update.service');
+
+module.exports = async (req, res) => {
+  try {
+    {
+      // validation
+      const expectedBody = Joi.object()
+        .required()
+        .keys({
+          name: Joi.string().required(),
+          email: Joi.string().required(),
+          phones: Joi.array().items(Joi.string().required())
+        });
+
```

Comment:
The validation schema is repeated in many places, could be moved to a validation util